src/expiring_hash_map.rs

```diff
@@ -12,8 +12,7 @@ use std::{
 use futures::StreamExt;
 use tokio_util::time::{delay_queue, DelayQueue};
 
-/// An expired item, holding the value and the key with an expiration
-/// information.
+/// An expired item, holding the value and the key with an expiration information.
```

Comment:
"an expiration information"?

Reply:
No idea. 🤷🏻 

I just needed to modify a Rust source file to trigger the CI step to run in the PR. Picked a random file.